gammapy/estimators/map/core.py

```diff
@@ -1093,6 +1105,28 @@ def read(cls, filename, checksum=False):
         ) as hdulist:
             return cls.from_hdulist(hdulist, checksum=checksum)
 
+    def copy(self, reference_model=None):
+        """Deep copy
+
+        Parameters
+        ----------
+
+        reference_model : `~gammapy.modeling.models.SkyModel`, optional
+            The reference model to use for conversions. If None, a model consisting
+            of a point source with a power law spectrum of index 2 is assumed.
+
+        Returns
+        -------
+        flux_maps : `FluxMaps`
+            Copied flux maps object.
+
+        """
+
+        new = deepcopy(self)
```

Comment:
Why are you not calling the copy function of Map here?

Reply:
The FluxMaps does not inherit of Map